### ISSUE TITLE
Packer build / checks failing for amazon linux 2

### ISSUE THREAD

**Issue by npalm** [philips-labs/terraform-aws-github-runner#3050]
See https://github.com/philips-labs/terraform-aws-github-runner/actions/runs/4387130262/jobs/7682047719

**Comment by scatybeneva** [comment]
From this [commit](https://github.com/philips-labs/terraform-aws-github-runner/commit/0d8a74cb2d6eff7e91b6a1e41a58d1e08f86965f)

you add in the startup-runner templatefile a check on metadata_tags. This works on userdata generated from terraform since it's defined within the terraform module. But all packer build use the same startup-runner template file but nowhere within packer variables is metadata_tags ever defined.

You also need to define [metadata_tags](https://github.com/philips-labs/terraform-aws-github-runner/blob/0d8a74cb2d6eff7e91b6a1e41a58d1e08f86965f/modules/runners/main.tf#L148-L150) https://github.com/philips-labs/terraform-aws-github-runner/blob/0d8a74cb2d6eff7e91b6a1e41a58d1e08f86965f/modules/runners/main.tf#L148-L150  from packer, perhaps a new var?

```
variable "metadata_options" {
  description = "Test retrieving tag information via AWS API (Cli)"
  type        = map(string)
  default     = {
    instance_metadata_tags      = "disabled"
    http_endpoint               = "enabled"
    http_tokens                 = "optional"
    http_put_response_hop_limit = 1
  }
}
```

**Comment by npalm** [comment]
Fixed in PR #3069
